conf/svcomp21.json

```diff
@@ -7,7 +7,10 @@
     "int": {
       "def_exc": true,
       "enums": false,
-      "interval": true
+      "interval": true,
+      "congruence": false,
+      "congruence_no_overflow": false,
+      "refinement": "never"
```

Comment:
Is there a reason to change our svcomp21 conf after the fact, even though these are the defaults anyway?